css/main.css

```diff
@@ -38,6 +39,78 @@ header {
   font-size: 18px;
   line-height: 24px;
 }
+/* Style the navigation menu */
+.topnav {
+  overflow: hidden;
+  background-color: #333;
+  /* position: relative; */
+  /* width: 100%;
+  background-color: red; */
```

Comment:
- [ ] I think it would be better to remove commented-out code because it keeps your codebase clean.  Refer to the first comment if you need further clarification 😄